util-java/src/com/liferay/util/servlet/GenericServletResponse.java

```diff
@@ -66,14 +66,21 @@ public void setContentLength(int length) {
 		_contentLength = length;
 	}
 
+	@Override
+	public void setContentLengthLong(long length) {
+		super.setContentLengthLong(length);
+
+		_contentLength = length;
+	}
+
```

Comment:
@holatuwol I am surprised that changing the return value from `int` to `long` for the `getContentLength()` method and the addition of the `setContentLengthLong(long)` method does not cause a MAJOR change in Semantic Versioning since package com.liferay.util.servlet is exported. It might be appropriate to annotate the class with `@ProviderType` just in case.